server/portal/apps/search/tasks.py

```diff
@@ -28,16 +33,15 @@ def agave_indexer(self, systemId, filePath='/', recurse=True, update_pems=False,
 
     try:
         filePath, folders, files = walk_levels(client, systemId, filePath, ignore_hidden=ignore_hidden).__next__()
+        index_level(filePath, folders, files, systemId, reindex=reindex)
```

Comment:
hmm should this remain outside the `try` block as well?

Reply:
Is this using the parameters from walk_levels?
If that's the case, it should be beneath that function?

Reply:
that looks right 👍